contract-ui/tabs/nfts/components/batch-lazy-mint.tsx

```diff
@@ -92,6 +87,8 @@ export const BatchLazyMint: React.FC<BatchLazyMintProps> = ({
 
   const mergedData = useMergedData(csvData, jsonData, imageFiles, videoFiles);
 
+  const invalidFiles = (csvData || jsonData) && mergedData.length === 0;
```

Comment:
I assume `mergedData.length` will be `0` in the case that there is no name field?

Reply:
yes ^